src/main/java/strategy/OperationStrategy.java

```
@@ -0,0 +1,7 @@
+package strategy;
+
+import model.FruitTransaction;
+
+public interface OperationStrategy {
+    OperationHandler getHandler(FruitTransaction.Operation operation);
```

Consider adding JavaDoc to `getHandler` to document the contract (for example: whether it should throw an informative exception when no handler is registered). The task/checklist expects that missing handlers are handled explicitly (the strategy should resolve a handler and throw if a handler is missing), so documenting this here makes the requirement clear for implementors .